pkg/url/url.go

```diff
@@ -12,12 +12,15 @@ import (
 	"github.com/redhat-developer/odo/pkg/util"
 
 	log "github.com/sirupsen/logrus"
+	"k8s.io/apimachinery/pkg/util/intstr"
+	"strings"
 )
 
 type URL struct {
 	Name     string
 	URL      string
 	Protocol string
+	Port     intstr.IntOrString
```

Comment:
Are we ever going to store anything else than a port number here? Wouldn't it be better to have this as `int`?